### PR TITLE
Start screen

### PR DESCRIPTION
**Pull request by mlytvynchuk**
Styled a start screen

### CODE REVIEW

File: app/src/main/java/com/google/droidjump/DroidStartView.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        super(context);
+    }
+
+    @Override
+    protected void onDraw(Canvas canvas) {
+        super.onDraw(canvas);
+        Bitmap fullDroidPic = decodeResource(getResources(), R.mipmap.droid);
+        int droidStep = (int) fullDroidPic.getWidth() / 9;
+        Bitmap droid = createBitmap(fullDroidPic, droidStep * 4, 0, droidStep,
+                fullDroidPic.getHeight(), null, true);
```
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/DroidStartView.java:38]: It is not obvious what is passed as "null" here. Please add a comment like:
```
Bitmap droid = createBitmap(fullDroidPic, droidStep * 4, 0, droidStep,
  fullDroidPic.getHeight(), /* parameterName= */ null, true);
```
where "parameterName" is the name of parameter in "createBitmap" function. 

Please do it for "0", "null", "true" and similar values where it is not obvious from context.

File: app/src/main/java/com/google/droidjump/HowToPlayFragment.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class HowToPlayFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+            Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.how_to_play_screen, container, false);
```
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/HowToPlayFragment.java:30]: Same as above about adding a parameter name comment

File: app/src/main/java/com/google/droidjump/DroidStartView.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Override
+    protected void onDraw(Canvas canvas) {
+        super.onDraw(canvas);
+        Bitmap fullDroidPic = decodeResource(getResources(), R.mipmap.droid);
+        int droidCountOnFullDroidPic = 9;
+        int droidStep = (int) fullDroidPic.getWidth() / droidCountOnFullDroidPic;
+        int droidCharacterStandingNumber = 4;
+        Bitmap droid = createBitmap(fullDroidPic, /* 4th droid on full droid picture */droidStep * droidCharacterStandingNumber, 0, droidStep,
```
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/DroidStartView.java:39]: What is this "0" for?

File: app/src/main/java/com/google/droidjump/DroidStartView.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                fullDroidPic.getHeight());
+        canvas.drawBitmap(droid, 0, getHeight() - droid.getHeight(), /* paint= */null);
```
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/DroidStartView.java:41]: Same - what is this "0" for?

File: app/src/main/java/com/google/droidjump/DroidStartView.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        super(context);
+    }
+
+    @Override
+    protected void onDraw(Canvas canvas) {
+        super.onDraw(canvas);
+        Bitmap fullDroidPic = decodeResource(getResources(), R.mipmap.droid);
+        int droidCountOnFullDroidPic = 9;
+        int droidStep = (int) fullDroidPic.getWidth() / droidCountOnFullDroidPic;
+        int droidCharacterStandingNumber = 4;
```
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/DroidStartView.java:38]: Could we create an enum/class with constants with these numbers like "droidCountOnFullDroidPic" and "droidCharacterStandingNumber"? I see that these values and other values are used in several of your pull requests.

File: app/src/main/java/com/google/droidjump/DroidStartView.java (hunk 3 above)
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/DroidStartView.java:39]: Make "droidStep * droidCharacterStandingNumber" a separate variable with a meaningful name, then you won't have to provide a comment like that.

Comment (mlytvynchuk) [reply app/src/main/java/com/google/droidjump/DroidStartView.java:39]: This is Y

File: app/src/main/java/com/google/droidjump/DroidStartView.java (hunk 5 above)
Comment (mlytvynchuk) [reply app/src/main/java/com/google/droidjump/DroidStartView.java:38]: I suggest using an interface that will be great here. What I found about constants in Java
https://www.informit.com/articles/article.aspx?p=331398&seqNum=5#:~:text=Java%20programmers%20commonly%20define%20constants,they%20are%20static%20and%20final.

**Pull request merged**